Classes/UVSuggestionListViewController.m

```diff
@@ -346,19 +269,29 @@ - (void)initNavigationItem {
     }
 }
 
+- (void)dismiss {
```

Comment:
@donnywdavis I just noticed that if you launch the forum directly it will not dismiss because you are overriding this method without calling super. Why is this needed here when we are already cleaning things up in dealloc?